deckhouse-controller/pkg/controller/module-controllers/release/controller.go

```diff
@@ -315,7 +316,7 @@ func (c *moduleReleaseReconciler) reconcileDeployedRelease(ctx context.Context,
 	}
 
 	// some other error apart from IsNotFound
-	if err != nil && !apierrors.IsNotFound(err) {
+	if !apierrors.IsNotFound(err) {
```

Comment:
Empty `err` will cast to the `StatusReasonUnknown` and IsNotFound(err) will return false, so, this condition will become `true` for empty errors. Isn't it?

Reply:
Yes, but on line 314 we already checked for nil. And so here we already have err!=nil